libraries/ssh_crypto.rb

```diff
@@ -174,4 +195,62 @@ def valid_privseparation
 
     ps
   end
+
+  # return a list of valid algoriths for a current platform
+  def valid_algorithms # rubocop:disable Metrics/CyclomaticComplexity, Metrics/MethodLength
+    alg53 = %w(rsa)
+    alg60 = %w(rsa ecdsa)
+    alg66 = %w(rsa ecdsa ed25519)
+    alg = alg66 # probably its a best suitable set for everything unknown
```

Comment:
here the question: should we expect a modern ssh version or not as fallback?

I did it with modern version, as I think we have covered all possible old distros and everything unknown should be modern.

Another view on this from somebody else?